static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -400,12 +407,7 @@ describe('WidgetBuilder', function () {
       // because of route setup
       renderTestComponent({params: {dashboardId: undefined}});
 
-      expect(await screen.findByRole('link', {name: 'Dashboard'})).toHaveAttribute(
```

Comment:
don't we want to check if this link exists?

Reply:
This is already being checked in the test above it on line 363

Reply:
I see... good catch!